src/shitalSDec19/assignment4_javaDecisionMakingPrograms/Solution6.java

```diff
@@ -0,0 +1,27 @@
+// 6.	Write a Java program that takes a year from user and 
+//print whether that year is a leap year or not
+
+
+package shitalSDec19.assignment4_javaDecisionMakingPrograms;
+
+import java.util.Scanner;
+
+public class Solution6 {
+	
+	void findLeapYear(int year)
+	{
+		if (year%4==0 && year%400==0)
```

Comment:
here, it should be year%4 && year%100 not year%4 && year%400
Do changes in next PR